packages/block-editor/src/private-apis.js

```diff
@@ -40,6 +40,7 @@ import {
 import { requiresWrapperOnCopy } from './components/writing-flow/utils';
 import { PrivateRichText } from './components/rich-text/';
 import { PrivateBlockPopover } from './components/block-popover';
+import useSpacingSizes from './components/spacing-sizes-control/hooks/use-spacing-sizes';
```

Comment:
The spacer block was using the list of presets to conditionally show UI elements. For the sake of not changing too many things to get this into 6.6, I just exported the same `useSpacingSizes` hook that the block editor package uses as a private API rather than refactoring the spacer block to not need it.